charts/rstudio-workbench/templates/tests/test-verify-installation.yaml

```diff
@@ -13,11 +13,12 @@ spec:
   {{- end }}
   shareProcessNamespace: {{ .Values.shareProcessNamespace }}
   restartPolicy: Never
+  {{- $topLevelParams := dict "enableDiagnostics" true "userCreate" true }}
+  {{- $disabledObject := dict "enabled" false }}
+  {{- $readinessProbe := dict "readinessProbe" $disabledObject }}
+  {{- $livenessProbe := dict "livenessProbe" $disabledObject }}
+  {{- $startupProbe := dict "startupProbe" $disabledObject }}
+  {{- $prometheusExporter := dict "prometheusExporter" $disabledObject }}
   {{- $overrideDict := . | deepCopy }}
-  {{- $_ := set $overrideDict.Values "enableDiagnostics" true }}
-  {{- $_ := set $overrideDict.Values "userCreate" true }}
-  {{- $_ := set $overrideDict.Values "readinessProbe.enabled" false }}
-  {{- $_ := set $overrideDict.Values "livenessProbe.enabled" false }}
-  {{- $_ := set $overrideDict.Values "startupProbe.enabled" false }}
-  {{- $_ := set $overrideDict.Values ".Values.prometheusExporter.enabled" false }}
+  {{- $_ := mergeOverwrite $overrideDict.Values $prometheusExporter $startupProbe $livenessProbe $readinessProbe $topLevelParams }}
```

Comment:
This looks awesome! Does this `$_` syntax work with `mergeOverwrite`? Should we Create a `Values {rest of things}` dict and set this output to `$overrideDict`?

Reply:
This works as is. `mergeOverwrite` actually overrides the sub values and thus pollutes the `overrideDict.Values`. 

Also, it is invalid syntax to do what you suggested (already tried something similar - Go templating won't let you do that).

Reply:
Ah cool! Good to know, thanks!! 🏆 TIL! 😄